pnpm-lock.yaml

```diff
@@ -4131,14 +3999,14 @@ packages:
     peerDependencies:
       react: ^18.3.1
 
-  react-dropzone@14.2.10:
-    resolution: {integrity: sha512-Y98LOCYxGO2jOFWREeKJlL7gbrHcOlTBp+9DCM1dh9XQ8+P/8ThhZT7kFb05C+bPcTXq/rixpU+5+LzwYrFLUw==}
+  react-dropzone@14.2.3:
+    resolution: {integrity: sha512-O3om8I+PkFKbxCukfIR3QAGftYXDZfOE2N1mr/7qebQJHs7U+/RSL/9xomJNpRg9kM5h9soQSdf0Gc7OHF5Fug==}
     engines: {node: '>= 10.13'}
     peerDependencies:
       react: '>= 16.8 || 18.0.0'
 
-  react-dropzone@14.2.3:
-    resolution: {integrity: sha512-O3om8I+PkFKbxCukfIR3QAGftYXDZfOE2N1mr/7qebQJHs7U+/RSL/9xomJNpRg9kM5h9soQSdf0Gc7OHF5Fug==}
+  react-dropzone@14.3.5:
```

Comment:
this could be it, don't know why this is also updated

Reply:
Probably because the lockfile was re-generated, that might be the reason. I am actually helping out the `react-dropzone` team in my spare time. If you have an idea of what is causing this I might be able to land a quick fix there.

Reply:
This might be related to https://github.com/react-dropzone/file-selector/issues/106 or https://github.com/cypress-io/cypress/issues/30629

Reply:
Do you really need to recreate the lock file this introduces a whole lot of new versions in a chaotic way. It will be better to make dependabot do these one at a time so we know what works and what does not

Reply:
Managed to fix this issue by providing a fix for the library, I'll resolve this thread.